spec/unit/filters/humanized_spec.rb

```diff
@@ -1,19 +1,31 @@
 require 'rails_helper'
 
 describe ActiveAdmin::Filters::Humanized do
-  let(:param) { ['category_id_eq', '1'] }
-
-  subject { ActiveAdmin::Filters::Humanized.new(param) }
-
   describe '#value' do
-    it 'should equal query string parameter' do
+    it 'should equal query string parameter if not an Array' do
+      param = ['category_id_eq', '1']
+      subject = ActiveAdmin::Filters::Humanized.new(param)
       expect(subject.value).to eq('1')
     end
+
+    it 'should equal query string parameters separated by commas if an Array' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.